go/vt/vtgate/planbuilder/testdata/unsupported_cases.txt

```diff
@@ -354,6 +349,7 @@ Gen4 plan same as above
 "unsupported: where clause for vindex function must be of the form id = <val> (rhs is not a value)"
 Gen4 plan same as above
 
+# vindex func without where condition
 "select keyspace_id from user_index"
 "unsupported: where clause for vindex function must be of the form id = <val> (where clause missing)"
 
```

Comment:
Gen4 does not produce a plan for this?

Reply:
It should be not supported

Reply:
We should produce the same error as v3 though